driver/main.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <reader.h>
+
+    using std::cout;
+    using std::cin;
+    using std::string;
+    using std::endl;
+
+    int main() {
+
+        string file_path;
+        cout << "Enter a config file path, if path unknown press enter." << endl;
+        getline(cin, file_path);
+
+        if(file_path == "") {
+            file_path = "C:\\Users\\Erik Thompson\\.CLion12\\system\\cmake\\generated\\819ddf16\\819ddf16\\Debug\\config.cfg";
+        }
+
+        config_reader::read_config(file_path);
```

Comment:
It would be nice if you received the return value and proved that your reader is working by looping through the values but otherwise you nailed it